hypertrace-trace-enricher/hypertrace-trace-enricher-impl/src/main/java/org/hypertrace/traceenricher/enrichment/enrichers/backend/AbstractBackendEntityEnricher.java

```diff
@@ -278,6 +278,11 @@ private Entity evaluateBackendEntity(Entity backendEntity) {
       Entity updatedEntity = mergeBackendEntity(existingEntity, backendEntity);
       if (!updatedEntity.equals(existingEntity)) {
         Entity result = this.upsertBackend(updatedEntity);
+        // update the cache to invalidate in case of update
+        entityCache
+            .getBackendIdAttrsToEntityCache()
+            .invalidate(
+                requestContext.buildContextualKey(backendEntity.getIdentifyingAttributesMap()));
         LOGGER.info("Updated backend:{}", result);
```

Comment:
We should convert this log into debug log, otherwise every update would lead an INFO log

Reply:
done